saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh

```diff
@@ -191,7 +191,11 @@ setup_ccmv2() {
   INVERTING_PROXY_FULL_URL="https://$INVERTING_PROXY_URL/"
 
   if [[ "$IS_PROXY_ENABLED" == "true" ]]; then
-    HTTP_PROXY_URL="http://$PROXY_HOST:$PROXY_PORT"
+    if [[ ! -z ${PROXY_USER} ]]; then
```

Comment:
you could use `-n` or just flip the branches so you could drop `!`

Reply:
thanks @lacikaaa . Did the change.